components/eventing-controller/controllers/subscriptionv2/jetstream/errors.go

```diff
@@ -0,0 +1,11 @@
+package jetstream
+
+import "github.com/pkg/errors"
+
+var (
+	errFailedToUpdateStatus       = errors.New("failed to update JetStream subscription status")
```

Comment:
Thank you for introduction of the new errors.

(nit)
in he jetstreamv2 backend we have the errors directly in the backend file. Here we have a dedicated file. IMO it's inconsistent. I suggest one of the following:
- Move the errors to the reconciler.go
- move the errors in the backend to a new errors.go file

what do you think ?

Reply:
The usage of these errors is also a good indicator of which errors were tested and which were not 😄